docs/source/migration.mdx

```diff
@@ -47,20 +40,157 @@ graph TB;
+- In Apollo Server 3, some framework integrations take a `path` option that lets you configure on what URL path they listen on, with a default of `/graphql`. This feature overlapped with the router features defined by web frameworks and has been removed in Apollo Server 4. This means that to provide the same API when migrating, you'll want to explicitly mount `expressMiddleware` at `/graphql`. (Note that the AS3 batteries-included server `apollo-server` listened on all paths, not just `/graphql`.)
+
+The follow sections show how servers using `apollo-server` and `apollo-server-express` need to change to use the new API.
+
+### Migrating from `apollo-server`
+
+If you used the batteries-included `apollo-server` package in Apollo Server 3, you'll want to use the `startStandaloneServer` function in Apollo Server 4. Call `startStandaloneServer(server)` instead of `server.listen()`. Note that you cannot configure CORS settings with `startStandaloneServer`; if you want to change CORS settings from the default, use `expressMiddleware` instead.
+
+<MultiCodeBlock>
+
+```ts title="apollo-server-3-standalone.ts"
+// npm install apollo-server graphql
+import { ApolloServer } from 'apollo-server';
+import { typeDefs, resolvers } from './schema';
+
+interface MyContext {
+  token?: String;
+}
+
+async function startApolloServer() {
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+    context: async ({ req }) => ({ token: req.headers.token }),
+  });
+  const { url } = await server.listen(4000);
+  console.log(`🚀  Server ready at ${url}`);
+}
+```
+
+</MultiCodeBlock>
+
+<MultiCodeBlock>
+
+```ts title="apollo-server-4-standalone.ts"
+// npm install @apollo/server graphql
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
+import { typeDefs, resolvers } from './schema';
+
+interface MyContext {
+  token?: String;
+}
+
+async function startApolloServer() {
+  const server = new ApolloServer<MyContext>({ typeDefs, resolvers });
+  const { url } = await startStandaloneServer(server, {
+    context: async ({ req }) => ({ token: req.headers.token }),
+    listen: { port: 4000 },
+  });
+  console.log(`🚀  Server ready at ${url}`);
+}
+```
+
+</MultiCodeBlock>
+
+
+### Migrating from `apollo-server-express`
+
+If you used the `apollo-server-express` package in Apollo Server 3, you'll want to use the `expressMiddleware` function in Apollo Server 4 instead of the `applyMiddleware` method. The changes include:
+
+- Installing `@apollo/server`, `cors`, and `body-parser` instead of `apollo-server-express` and `apollo-server-core`
+- Importing symbols from `@apollo/server` instead of `apollo-server-express` and `apollo-server-core`
+- Explicitly adding `cors` and `bodyParser.json()
+
+Note that by default, `apollo-server-express` in AS3 would only actually respond to operations on `/graphql`, so we explicitly mount `expressMiddleware` at that path in AS4.
```

Comment:
Actually maybe it should go in the bulleted list above instead?

Reply:
Oooh yes! I love that, so something like:
```suggestion
- If using `apollo-server-express`'s default [`/graphql` URL path](apollo-server/api/apollo-server/#path), explicitly mount `expressMiddleware` at `/graphql`. 
```

Reply:
That sounds good. Maybe get in somewhere "instead of passing the `path` option to `applyMiddleware` or `getMiddleware`" to be clear about what you're looking for to see if you're doing this?

Or alternatively, say something like "if you pass the `path` option then you just need to pass it directly to `app.use` instead; if you don't pass the `path` option then to avoid behavior change you should pass `/graphql` to `app.use` since that is the AS3 default"?